device-types/NEC/UNIVERGE-IX2207.yaml

```diff
@@ -0,0 +1,37 @@
+---
+manufacturer: NEC
+model: UNIVERGE IX2207
+slug: univerge-ix2207
+part_number: BI000059
+comments: |
+  NEC UNIVERGE IX2207, VPN-Compatible High speed Access Router, 24W
+  (1) RJ-45 Console Port, (2) Ethernet 1000BASE-T, (4) Ethernet 1000BASE-T in Switching Hub, (2) USB2.0 Type-A
+
+  Dimentions(WxDxH in mm): 210x148x43
+u_height: 1
+is_full_depth: false
+console-ports:
+  - name: CONSOLE
+    type: rj-45
+interfaces:
+  - name: GigaEthernet0
+    type: 1000base-t
+  - name: GigaEthernet1
+    type: 1000base-t
+  - name: GigaEthernet2 port 1
+    type: 1000base-t
+  - name: GigaEthernet2 port 2
+    type: 1000base-t
+  - name: GigaEthernet2 port 3
+    type: 1000base-t
+  - name: GigaEthernet2 port 4
+    type: 1000base-t
+  - name: USB0
```

Comment:
Are these USB ports just to transfer data from a USB? If so, please remove these as they are not meant to be modelled like this

Reply:
Data transfer is also possible, but the main purpose is to connect a USB LTE modem.